### ISSUE TITLE
Saved figure does cut out the empty area

### ISSUE THREAD

**Issue by boxcwang** [aplpy/aplpy#375]
Hi,

We just updated matplotlib and aplpy to the latest version,
and I notice that my old plotting script now could not save the figure properly.

I tried with: 
fig.save(figname,transparent=False,adjust_bbox=tight,dpi=600)
and 
pylab.savefig(figname,transparent=False,bbox_inches='tight',dpi=600),

The saved figure seams ignored my figure size set up and did not cut out the 
empty area properly.

The script worked perfectly under the old version,
matplotlib 2.0, aplpy 0.9.12.

Would you please help me to solve this?

Thank you in advance.

Cheers

Yuan

**Comment by astrofrog** [comment]
@boxcwang - can you confirm which version of APLpy you are using? For example:

```python
In [1]: from aplpy import __version__
In [2]: print(__version__)
1.1.1
```

**Comment by boxcwang** [comment]
Yes, it is 1.1.1

In [1]: from aplpy import __version__
/usr/lib64/python2.7/site-packages/matplotlib/cbook/deprecation.py:106: MatplotlibDeprecationWarning: The mpl_toolkits.axes_grid module was deprecated in version 2.1. Use mpl_toolkits.axes_grid1 and mpl_toolkits.axisartist provies the same functionality instead.
  warnings.warn(message, mplDeprecation, stacklevel=1)

In [2]: print(__version__)
   ...:
1.1.1

**Comment by boxcwang** [comment]
BTW, we have matplotlib 2.1.2 currently.
In [4]: matplotlib.__version__
Out[4]: '2.1.2'

Cheers

Yuan

**Comment by astrofrog** [comment]
@boxcwang - can you include the figure in a comment here so I can see what you mean by not being cut out correctly?

**Comment by boxcwang** [comment]
Sure.
This is what the saved figure looks like before the update:
<img width="563" alt="screen shot 2018-02-10 at 23 56 31" src="https://user-images.githubusercontent.com/25138336/36067460-156120c6-0ebe-11e8-9c3a-eb49726e289c.png">

This is what it saved with the same script but newest aplpy and matplotlib.
<img width="495" alt="screen shot 2018-02-10 at 23 59 04" src="https://user-images.githubusercontent.com/25138336/36067478-668e7ffc-0ebe-11e8-9c81-1e86a6108da6.png">

Although now I have found a solution to save the image with the proper size.
I still do not know why exactly the same script produced so different image.

Cheers

Yuan

**Comment by astrofrog** [comment]
@boxcwang - thanks! Can you show me the full script you are using?

**Comment by boxcwang** [comment]
Sure, here it is:

import matplotlib,pylab,aplpy

params={'figure.figsize': figsize,
        'text.usetex': True,
        'axes.labelsize': 9,
        'font.size': 9,
        'xtick.major.size':1.2,
        'xtick.minor.size':0.6,
        'ytick.major.size':1.2,
        'ytick.minor.size':0.6,
        'xtick.labelsize': 9,
        'ytick.labelsize': 9,
        'legend.fontsize': 4.5,
        'xtick.major.pad': 1,
        'ytick.major.pad': 0.8,}

pylab.rcParams.update(params)

filename='Cont_1400_mosaic_all_ft_vgps_5_cut.fits'
figname='Cont_1400_mosaic_all_ft_vgps-log-5.eps'

fig=aplpy.FITSFigure(filename)
fig.show_colorscale(vmin=0.008,stretch='log',vmax=0.1, cmap='RdBu_r')

fig.frame.set_color('black')
fig.frame.set_linewidth(0.1)
fig.ticks.set_color('black')


fig.tick_labels.set_xformat('ddd')
fig.tick_labels.set_yformat('dd')
fig.ticks.set_xspacing(1.0)
fig.ticks.set_yspacing(1.0)
fig.ticks.set_length(6)
fig.ticks.set_linewidth(0.3)
fig.axis_labels.set_xtext('Galactic Longitude')
fig.axis_labels.set_ytext('Galactic Latitude')
fig.tick_labels.set_yposition('left')
fig.tick_labels.set_xposition('bottom')
fig.axis_labels.set_xpad(0.03)
fig.axis_labels.set_ypad(0.03)
fig.axis_labels.set_yposition('left')
fig.axis_labels.set_xposition('bottom')

fig.add_colorbar()
fig.colorbar.set_width(0.06)
fig.colorbar.set_pad(0.01)
fig.colorbar.set_frame_linewidth(0.2)
fig.colorbar.set_axis_label_text('Jy beam$^{-1}$')
fig.colorbar.set_axis_label_rotation('vertical')
fig.colorbar.set_axis_label_pad(-0.02)
fig.colorbar.set_font(size=7.5)

fig.save(figname,transparent=False,adjust_bbox=True,dpi=600)
fig.close()



Cheers

Yuan

**Comment by astrofrog** [comment]
@boxcwang - it looks like this should be fixed in APLpy 2.0 - please re-open this issue if not!
